src/StorageSync/StorageSync/StorageSync.csproj

```diff
@@ -10,6 +10,13 @@
     <RootNamespace>$(LegacyAssemblyPrefix)$(PsModuleName)</RootNamespace>
   </PropertyGroup>
 
+  
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.StorageSync" Version="2.8.0-preview" />
+    <PackageReference Include="Microsoft.Win32.Registry" Version="4.5.0" />
```

Comment:
Please suggest a way to repro the issue this change would cause. Also, Please suggest the recommended change.

---
In reply to: [256618122](https://github.com/Azure/azure-powershell/pull/8527#discussion_r256618122) [](ancestors = 256618122)